Tour-Collective-Devs/frontend-jobsite
# Rm current users profile

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes # not completed ticket yet, progress towards closing ticket 3, user profile

## Description of Proposed Changes
Added basic logic to render user profile based on their profile type

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout rm-current-users-profile
```
1. load page
1. go to profile, by default a crew member profile should be loaded
1. go into UserProfile.js and change the state to be an employer profile, an employer profile should be loaded.
1. go into code again and change code to something else, an error page should be loaded
1. the information in UserProfile.js state is test code. this information will eventually come from external api, most likely through the context provider.

## Impacted Areas in Application
Current User Profile

List general components of the application that this PR will affect:

## Mentions @username

Tag users that need to review this code

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
1. Every class must be documented with purpose, author, and methods.
1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
